RadixWallet/Features/AccountAndPersonaHidingFeature/AccountAndPersonaHiding+Reducer.swift

```diff
@@ -0,0 +1,81 @@
+// MARK: - AccountAndPersonaHiding
+public struct AccountAndPersonaHiding: FeatureReducer {
+	public struct State: Hashable, Sendable {
+		public var hiddenEntitiesStats: EntitiesVisibilityClient.HiddenEntitiesStats?
+
+		@PresentationState
+		public var confirmUnhideAllAlert: AlertState<ViewAction.ConfirmUnhideAllAlert>?
+	}
+
+	public enum ViewAction: Hashable, Sendable {
+		case task
+		case unhideAllTapped
+
+		case confirmUnhideAllAlert(PresentationAction<ConfirmUnhideAllAlert>)
+
+		public enum ConfirmUnhideAllAlert: Hashable, Sendable {
+			case confirmTapped
+			case cancelTapped
+		}
+	}
+
+	public enum InternalAction: Hashable, Sendable {
+		case hiddenEntitesStatsLoaded(EntitiesVisibilityClient.HiddenEntitiesStats)
+		case didUnhideAllEntities
+	}
+
+	@Dependency(\.entitiesVisibilityClient) var entitiesVisibilityClient
+	@Dependency(\.overlayWindowClient) var overlayWindowClient
+	@Dependency(\.errorQueue) var errorQueue
+
+	public func reduce(into state: inout State, viewAction: ViewAction) -> Effect<Action> {
+		switch viewAction {
+		case .task:
+			return .run { send in
+				let hiddenEntitiesStats = try await entitiesVisibilityClient.getHiddenEntitiesStats()
+				await send(.internal(.hiddenEntitesStatsLoaded(hiddenEntitiesStats)))
+			}
+		case .unhideAllTapped:
+			state.confirmUnhideAllAlert = .init(
+				title: .init(L10n.AppSettings.EntityHiding.unhideAllSection),
+				message: .init(L10n.AppSettings.EntityHiding.unhideAllConfirmation),
```

Comment:
I suppose the image should also be called "entity-hiding"?

Reply:
yeah, I think it just uses the name of the resource I did download, will update.